.travis.yml

```diff
@@ -26,12 +26,12 @@ deploy:
 install:
   - sudo apt-get install gcc libaspell-dev libaspell15 aspell
 jdk:
-  - openjdk7
-  - oraclejdk8
+  - openjdk8
+dist: bionic
 
 language: scala
 scala:
-  - 2.10.6
-  - 2.11.7
+  - 2.10.7
+  - 2.11.12
```

Comment:
should we add 2.12.10 to this list?

Reply:
Eventually, yes. I want to get the builds passing on existing Scala versions before expanding to new verisons.